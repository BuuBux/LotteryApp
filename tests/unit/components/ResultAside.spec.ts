import ResultAside from '@/components/ResultAsideComponent.vue';
import { mount } from '@vue/test-utils';
import testSetup from '../testSetup';


const propsData = { type: 'eurojackpot' };

const createWrapper = () => mount(ResultAside, {
  ...testSetup,
  propsData,
});

describe('Testing ResultAsideComponent.vue', () => {

  it('Create a snapshot', () => {
    expect(createWrapper()).toMatchSnapshot();
  });

  it('Check is selected date is string and not null', () => {
    const wrapper = createWrapper();
    expect(wrapper.props().type).toBe('eurojackpot');
  });
});
