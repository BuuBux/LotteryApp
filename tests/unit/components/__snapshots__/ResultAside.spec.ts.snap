// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing HomeContentComponent.vue Create a snapshot 1`] = `
<div class="c-dates__wrapper">
  <div class="c-panel">
    <h2 class="c-dates__header"> Check results for other dates </h2>
    <!---->
    <!---->
    <div class="c-dates__buttons"><button class="c-btn">
        Load more dates
      </button></div>
  </div>
</div>
`;
